src/applications/personalization/profile360/components/REMOVAL_PROFILE_NOTES

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
👍

Reply:
I will remove this file once I'm ready to merge step 3, this just helps me keep track of what's going on with all these files I'm either removing or keeping 😄